src/odemis/util/__init__.py

```diff
@@ -844,3 +845,35 @@ def project_point_on_line(
         x_projected = (point[0] + line_slope * (point[1] - line_intercept)) / (1 + line_slope**2)
         y_projected = line_slope * x_projected + line_intercept
     return (x_projected, y_projected)
+
+
+def read_json(file_path: str) -> Optional[Any]:
+    """
+    Read a json file.
+
+    :param file_path: The absolute file path of the json file.
+
+    """
+    try:
+        with open(file_path, "r") as file:
+            data = json.load(file)
+        return data
+    except FileNotFoundError:
+        logging.info("The json file at %s was not found.", file_path)
+    except Exception:
+        logging.exception("Failed to load json file at %s", file_path)
+
+
+def write_json(file_path: str, data: Any):
```

Comment:
doesn't it make more sense to have these functions somewhere in `odemis.dataio` or otherwise in `odemis.util.dataio`?